pages/examples/features/authentication/bluesky.tsx

```diff
@@ -74,15 +69,25 @@ function ExampleAuthentication(props) {
     >
       <KeyHeader onInputChange={setKey} value={key} viewer={currentUser} />
       <ThinAppLayout>
-        <FormHeading style={{ marginTop: 64 }}>Sign in with Bluesky [WIP]</FormHeading>
+        <FormHeading style={{ marginTop: 64 }}>Sign in with Bluesky</FormHeading>
         <FormParagraph>
           Waiting for Paul Frazee to finish the Bluesky OAuth implementation he promised me with route redirects and all. Once that is complete, you will be able to sign in with
```

Comment:
@BinaryFiddler 

The Bluesky team has implemented a new OAuth scope, transition:email that clients can request for access to the account's email so now our API supports authentication through Bluesky the same way Google OAuth would work. The email itself can be accessed via the com.atproto.server.getSession endpoint on the PDS, using an OAuth access token. This is a living an example of this method.